src/plugins/wizard/public/application/utils/get_top_nav_config.tsx

```diff
@@ -50,8 +50,20 @@ interface TopNavConfigParams {
 
 export const getTopNavConfig = (
   { visualizationIdFromUrl, savedWizardVis, saveDisabledReason, dispatch }: TopNavConfigParams,
-  { history, toastNotifications, i18n: { Context: I18nContext } }: WizardServices
+  services: WizardServices
 ) => {
+  const {
+    i18n: { Context: I18nContext },
+    embeddable,
+    scopedHistory,
+  } = services;
+
+  const { originatingApp: originatingApp } =
```

Comment:
nit: why are you renaming it to the same variable name again?

Reply:
yea sry i forgot to change the naming, will just use originatingApp